src/components/VideoPlayer/index.js

```diff
@@ -0,0 +1,184 @@
+import { useRef, useReducer, useEffect, useState } from "react";
+import classes from "./VideoPlayer.module.css";
+
+const PlayerStateManager = (state, action) => {
+  switch (action.type) {
+    case "SETTINGS":
+      return { ...state, settings: !state.settings };
+    case "PLAYBACK":
+      return { ...state, playback: !state.playback };
+    case "CONTROLLER":
+      return { ...state, controller: !state.controller };
+    case "PLAY":
+      return { ...state, play: action.payload };
+    case "RESET":
+      return { ...state, controller: false, settings: false, playback: false };
+    default:
+      return state;
+  }
+};
+
+const VideoPlayer = ({ src, autoPlayTrue }) => {
+  const videoRef = useRef();
+  const progressBar = useRef();
+  const [progress, setProgress] = useState(0);
+
+  const [state, dispatch] = useReducer(PlayerStateManager, {
+    controller: false,
+    settings: false,
+    playback: false,
+    play: false,
+  });
+
+  const forwardVideoBy10 = () => {
+    if (videoRef.current.currentTime !== videoRef.current.duration) {
+      videoRef.current.currentTime += 10;
+      dispatch({ type: "PLAY", payload: true });
+    }
+  };
+  const backwardVideoBy10 = () => {
+    if (videoRef.current.currentTime !== 0) {
+      videoRef.current.currentTime -= 10;
+      dispatch({ type: "PLAY", payload: true });
+    }
+  };
```

Comment:
This is great. You have implemented Forward and Backward functionality. Great work.